pkg/transformer/tlp.go

```diff
@@ -160,3 +155,70 @@ func partition(expr ast.ExprNode) []ast.ExprNode {
 func RandTLPType() TLPType {
 	return TLPTypes[util.Rd(len(TLPTypes))]
 }
+
+func tryAggTransform(selectStmt *ast.SelectStmt, unionStmt *ast.UnionStmt) (ast.ResultSetNode, error) {
+	if selectStmt.Fields != nil && len(selectStmt.Fields.Fields) != 0 {
+		aggFns := make([]int, 0)
+		exprNames := make(map[string]bool)
+		selectFields := make([]*ast.SelectField, 0, len(selectStmt.Fields.Fields))
+		unionFields := make([]*ast.SelectField, 0, len(selectStmt.Fields.Fields))
+		for index, field := range selectStmt.Fields.Fields {
+			selectField, unionField := *field, *field
+			selectFields = append(selectFields, &selectField)
+			unionFields = append(unionFields, &unionField)
+			if !field.Auxiliary {
+				if field.AsName.String() != "" {
+					exprNames[field.AsName.String()] = true
+				}
+				if fn, ok := field.Expr.(*ast.AggregateFuncExpr); ok {
```

Comment:
How about normal function composed with aggregation function? Like `SELECT 100+sum(col) FROM ...`

Reply:
`100+sum(col)` is not self-composable